packages/manager/apps/pci-instances/src/components/menu/ActionsMenu.component.tsx

```diff
@@ -10,57 +7,22 @@ import {
   Divider,
   Icon,
   ICON_NAME,
-  Link,
   Popover,
   PopoverContent,
   PopoverTrigger,
 } from '@ovhcloud/ods-react';
 import { DeepReadonly } from '@/types/utils.type';
-
-export type TActionsMenuItem = DeepReadonly<{
-  label: string;
-  link: {
-    path: string;
-    isExternal: boolean;
-  };
-}>;
+import { ActionMenuItem, TActionsMenuItem } from './ActionMenuItem.component';
 
 export type TActionsMenuProps = DeepReadonly<{
-  items: Map<string, TActionsMenuItem[]>;
+  items: Map<string, TActionsMenuItem[]> | TActionsMenuItem[];
```

Comment:
I'm not very confident having this kind of union type as items Map and Array have not the same structure that causes to have multiple weak assertions inside the logic.

Could we stick the dashboard usage as a `Map`, in the same way we did for data grid actions?

Reply:
And for the caller it's just a one element map containing the array